testutil/kubernetes_helper.go

```diff
@@ -82,7 +82,7 @@ func (h *KubernetesHelper) createNamespaceIfNotExists(namespace string, annotati
 
 // CreateControlPlaneNamespaceIfNotExists creates linkerd control plane namespace.
 func (h *KubernetesHelper) CreateControlPlaneNamespaceIfNotExists(namespace string) error {
-	labels := map[string]string{"linkerd.io/is-control-plane": "true", "config.linkerd.io/admission-webhooks": "disabled"}
+	labels := map[string]string{"linkerd.io/is-control-plane": "true", "config.linkerd.io/admission-webhooks": "disabled", "linkerd.io/control-plane-ns": "linkerd"}
```

Comment:
Seems to me here we need to use the `namespace` variable for the name of the namespace :)

Reply:
Done!